fuse_loss/src/tukey_loss.cpp

```diff
@@ -33,6 +33,8 @@
  */
 #include <fuse_loss/tukey_loss.h>
 
+#include <fuse_core/ceres_options.h>
```

Comment:
This headers has the `CERES_VERSION_AT_LEAST` macro, but also a bunch of Ceres `Options` classes that we don't need here.

I wonder if I should create a new header with the macros only, something like: `fuse_core/ceres_macros.h` :thinking:

Reply:
It might be nice to separate out the macros. As it is, you are pulling in some ROS dependencies that are not technically required. That said, I'm not too concerned about it right now. This is a ROS package that uses Ceres; both those dependencies will be available.

Reply:
Just added a new `fuse_core/ceres_macros.h` header. Not a big deal, and it reduces "surprise" when someone sees the header that is including here, because we aren't using any Ceres `Options` here.